public/app/features/alerting/unified/rule-list/components/ListItem.tsx

```diff
@@ -31,7 +31,7 @@ export const ListItem = (props: ListItemProps) => {
         {/* icon */}
         <span className={styles.statusIcon}>{icon}</span>
 
-        <Stack direction="column" gap={0} flex="1" minWidth={0}>
+        <Stack direction="column" gap={0.5} flex="1" minWidth={0}>
```

Comment:
For some reason the description adds its own paddings, so if we set this to 0 the line heights wouldn't be balanced out.